.travis.yml

```diff
@@ -17,6 +17,7 @@ install:
   # Project configuration requirements.
   - pip install virtualenv virtualenvwrapper
   - pip install autoenv
+  - pip install tox
   - git clone https://github.com/cfpb/sheer
   - pip install -e sheer
   - pip install -r sheer/requirements.txt
```

Comment:
There should be a `git clone https://github.com/cfpb/django-sheerlike` and `pip install -e django-sheerlike` since that's what is going to power the "sheerlike" part of the site until we make the conversion.